client/my-sites/domains/domain-search/index.jsx

```diff
@@ -277,7 +277,10 @@ class DomainSearch extends Component {
 						<div className="domains__header">
 							{ domainSidebarExperimentUser && (
 								<>
-									<DomainAndPlanPackageNavigation step={ 1 } />
+									<DomainAndPlanPackageNavigation
+										goBackLink={ domainManagementList( selectedSiteSlug ) }
```

Comment:
I didn't find a reliable way to get the previous history from the user, so used the same one we already use on the domains page.